backend/order/mails.py

```diff
@@ -70,3 +70,37 @@ def send_mail_on_create_order_user(order, products):
         html_message=html_message,
         fail_silently=True,
     )
+
+
+def send_main_on_cancel_order(order):
+    title = f"Cancelación orden de compra - {order.pk} - Socio { order.user } "
+    subject = f"GAQSA - {title}"
+    provider = Provider.objects.get(pk=order.provider.pk)
+    user = provider.user
+    context = {
+        "pk": order.pk,
+        "user": user,
+        "client": order.user,
+        "title": title,
+    }
+    from_email = "noreply@gaqsa.com"
+
+    provider_emails = list(UserEmail.objects.filter(
+        user=user, category=UserEmail.ORDERS
+    ).values_list('email', flat=True))
+    from_email = "noreply@gaqsa.com"
+    # TODO: Cambiar correo de admin
+    to_emails = provider_emails + ["temp@temp.com"]
```

Comment:
Podemos utilizar la función de utils para obtener todos los correos de los admins y no dejarlo hardcodeado.

```suggestion
from backend.utils.emails import get_admin_emails
    admin_emails = get_admin_emails()
    to_emails = provider_emails + admin_emails
```